src/sentry/south_migrations/0494_auto__del_field_eventattachment_group_id.py

```diff
@@ -0,0 +1,1454 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
How high volume is this table? Should this be a dangerous migration?

Reply:
I don't think it's high (this feature is currently turned on for only 16 orgs)

Reply:
@JTCunning / @mattrobenolt Do you have any preference here around doing this manually?

Reply:
Double checked this with Matt, going to leave as False